iop/fs/bdmfs_fatfs/src/fs_driver.c

```diff
@@ -681,6 +707,18 @@ int fs_ioctl2(iop_file_t *fd, int cmd, void *data, unsigned int datalen, void *r
         case USBMASS_IOCTL_GET_FRAGLIST:
             ret = get_frag_list(file, rdata, rdatalen);
             break;
+        case BDM_IOCTL_GET_DEVICE_INDEX:
+        case BDM_IOCTL_GET_LBA_BITS:
+        {
+            struct block_device *mounted_bd = fatfs_fs_driver_get_mounted_bd_from_index(fd->unit);
+            if (mounted_bd == NULL)
+                ret = -ENXIO;
+            else if (mounted_bd->ioctl == NULL)
+                ret = -ENOTSUP;
+            else
+                ret = mounted_bd->ioctl(mounted_bd, cmd, data, datalen, rdata, rdatalen);
+            break;
+        }
```

Comment:
Why is it needed to know the 'bits' that are supported by the driver and/or the interface?

The number of bits needed to address the block device can also be extracted from the size.

Reply:
Explained in the comment about why add an ioctl interface to bdm: this is used on the EE side to inform OPL cdvdman which device command set to use for proper device access.